### PR TITLE
Add new reference for MaxQuant, MSStats and MSStatsTMT

### PR DESCRIPTION
**Pull request by npinter**
I've added https://pubs.acs.org/doi/10.1021/acs.jproteome.2c00051 to the references (and did some small changes regarding the version suffix in MSStats and MSStatsTMT).

### CODE REVIEW

File: tools/maxquant/macros.xml [hunk 1]
```diff
@@ -882,6 +882,7 @@
                 }
             </citation>
             <citation type="doi">10.1021/acs.jproteome.5b00780</citation>
```
Comment (bgruening) [inline tools/maxquant/macros.xml:884]: should the old one be dropped?

Comment (npinter) [reply tools/maxquant/macros.xml:884]: @foellmelanie what do you think? keep both?

Comment (npinter) [reply tools/maxquant/macros.xml:884]: @foellmelanie has agreed to leave both references in!

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks.

**Pull request merged**